### DISCUSSION TITLE
I'm back! + Update + New app???

### DISCUSSION THREAD

**Discussion by Jeffser** [Jeffser/Alpaca#370] [Announcements]
> [!NOTE]
> This is an update of [this discussion](https://github.com/Jeffser/Alpaca/discussions/361)

Hi everyone it's me!

You might know that I had to go through surgery last week, I've been doing fine but it seems like it's going to take a while for me to fully recover. Now, I can't just disappear for months from my open source projects so I decided to start coding with my laptop in bed.

So yeah, I'm back I guess...

## What's Next

So I wanted to announce what I was working on before the whole surgery thing, it's called **Sunset**, it is a Bluesky client that uses Adwaita, since a bunch of people stopped using Twitter and switched over to Bluesky, I wanted to do something nice for the Linux community!

Now, this is just a small prototype with really awful code, I'm just learning how the ATProtocol works and how I can implement it with some custom GTK widgets.

The project will be completely open source and available on Flathub and all that good stuff but for now the repository is private.

I uploaded a couple of images and a video to [my account on Bluesky](https://bsky.app/profile/jeffser.com), I will probably keep updating you guys on the development there so please go check it out.

I will probably focus on developing Sunset instead of Alpaca since it's admittedly more fun to start a new project than to add small features to the existing one (not gonna abandon it tho)

So yeah, thanks for the support and all the good wishes on the last post. I hope you all had a nice Halloween!

![image](https://github.com/user-attachments/assets/9604193b-9c01-45b2-bc34-5b36801b831a)

**Comment by olumolu** [comment]
Is it possible to have mastodon support also

**Reply by Jeffser** [reply]
That's a completely different thing, you should use [Tuba!](https://flathub.org/apps/dev.geopjr.Tuba)
